cdap-ui/app/cdap/services/WizardStores/AddNamespace/AddNamespaceStore.js

```diff
@@ -30,28 +30,28 @@ const defaultGeneralState = Object.assign({
   name: '',
   description: '',
   schedulerQueue: ''
-}, defaultState);
+}, defaultState, {__skipped: false});
 
 const defaultMappingState = Object.assign({
   hdfsDirectory: '',
   hiveDatabaseName: '',
   hbaseNamespace: ''
-}, defaultState);
+}, defaultState, {__complete: true, __skipped: true});
```

Comment:
I don't know if we are actually using the entire defaultState anymore here? Why not have a `skippableDefaultState` and a `nonSkippableDefaultState`?